sdk/react/widgets/ToggleSwitch.js

```diff
@@ -24,7 +25,18 @@ var ToggleSwitch = React.createClass({
     switchOnText: React.PropTypes.string,
     switchOffText: React.PropTypes.string,
     onValueChanged: React.PropTypes.func,
-    config: React.PropTypes.object
+    config: React.PropTypes.object,
+    onTintColor: React.PropTypes.string,
+    tintColor: React.PropTypes.string,
+    thumbTintColor: React.PropTypes.string,
+  },
+
+  getDefaultProps: function() {
+    return {
+      onTintColor: '#498DFC',
```

Comment:
okay, so our SDK does not support the change of those values yet by the customer.  That's fine, I just wanted to understand that.

Thanks,
Mike